omod/src/main/webapp/resources/scripts/navigator/validators.js

```diff
@@ -156,13 +156,27 @@ RegexFieldValidator.prototype.validate = function(field) {
     return null;
 }
 
+function PhoneFieldValidator() {
```

Comment:
This looks fine but i think adding a regex class to the element and specifying the regex could have done it